docs/en/guides/30-sql-clients/00-bendsql/index.md

```diff
@@ -66,9 +66,15 @@ When building from source, some dependencies may involve compiling C/C++ code. E
 cargo install bendsql
 ```
 
+## User Authentication
+
+If you are connecting to a self-hosted Databend instance, you can use the admin users specified in the [databend-query.toml](https://github.com/datafuselabs/databend/blob/main/scripts/distribution/configs/databend-query.toml) configuration file, or you can connect using an SQL user created with the [CREATE USER](/sql/sql-commands/ddl/user/user-create-user) command. 
+
+For connections to Databend Cloud, you can use the default `cloudapp` user or an SQL user created with the [CREATE USER](/sql/sql-commands/ddl/user/user-create-user) command. Please note that the user account (email address) you use to log in to the Databend Cloud console cannot be used for connections. However, you can grant roles or privileges to users based on their email addresses. These user accounts, identified by their email addresses, can also function as SQL users in Databend Cloud. For more information, see [Members](/guides/cloud/manage/members).
```

Comment:
```
Please note that the user account (email address) you use to log in to the Databend Cloud console cannot be used for connections. However, you can grant roles or privileges to users based on their email addresses.
```
This parts need @flaneur2020 to have a check.

Reply:
how about "Please note that you can not directly use your Databend Cloud account to connect the Databend services, but you could manage your Databend users in Databend Cloud console with CREATE USER statement." ?

Reply:
Make the separation between the inner service's user/password and the outer console's account might be easier to understand.

also, the Databend Cloud account might not be tied to an email/password but could also be integrated with SSO (Single Sign-On) like GitHub Auth.

Reply:
For connections to Databend Cloud, you can use the default cloudapp user or an SQL user created with the [CREATE USER](http://localhost:3000/sql/sql-commands/ddl/user/user-create-user) command. Please note that the user account you use to log in to the [Databend Cloud console](https://app.databend.com/) cannot be used for connecting to Databend Cloud.